build_patch/libkrw/tfp0.patch

```diff
@@ -1,14 +1,25 @@
-diff --git a/src/libkrw.c b/src/libkrw.c
-index 4c0e50a..5ad473d 100644
---- a/src/libkrw.c
-+++ b/src/libkrw.c
-@@ -107,9 +107,7 @@ static void iterate_plugins(int (*callback)(void *), void **check) {
+--- a/src/libkrw.c	2024-04-20 14:46:12.796173476 -0700
++++ b/src/libkrw.c	2024-04-20 18:01:14.500961452 -0700
+@@ -160,3 +160,8 @@
+     if (krw_handlers.physwrite == NULL) return ENOTSUP;
+     return krw_handlers.physwrite(from, to, len, granule);
  }
++
++__attribute__((visibility("hidden")))
```

Comment:
Why declare this function at all? Remove it and its usage entirely.
You may want to fork libkrw though.

Reply:
The original code has it as hidden. Meh I didn't really change anything codewise so I doesn't need a fork imo.